### PR TITLE
Brings back the old scrubber/vent sprites.

### PR DESCRIPTION
**Pull request by praisenarsie**
:cl: Frozenguy5
imageadd: Brought back the old vent/scrubber sprites.
/:cl:

[why]: They seem out of place and unnatural plus they are too eyecandy meaning it's really easy to notice them. Plus they are too big, etc, etc. I do like change but I am not fond of these sprites as I have just stated the reasons why.

hopefully this should work

### CODE REVIEW

**Comment by BeeSting12** [comment]
These new scrubbers are one of the few sprite changes I actually like. At very worse I feel like it's a sidegrade tbh.

**Comment by praisenarsie** [comment]
icondiffbot is high so...



compared to

**Comment by praisenarsie** [comment]
@BeeSting12 
I understand what you are getting at but one problem I find with these is the siphon animation is hard to notice when old scrubbers were really easy to tell if it was siphoning to be honest.

**Comment by vuonojenmustaturska** [comment]
It should be added that the new sprites were work-in-progress in the codebase they were taken from

**Comment by zamolxius** [comment]
Thank you a lot

**Comment by uraniummeltdown** [comment]
Thank you the current ones looked awful and stuck out like a sore thumb

**Comment by calzilla1** [comment]
Can we remove the directional computers too?

**Comment by deathride58** [comment]
The new sprites are actually pretty good. The scrubbers in particular aren't abstract, and the vents no longer show their status by being a flashing beacon of light.

Also
>They're too big
The sprites are the same size. Only difference is that they're off-center, which can be fixed.

**Comment by praisenarsie** [comment]
they are like 1-2 pixels bigger.

**Comment by lzimann** [comment]
That diff should be correct now(in case it isn't, let me know).

**Comment by Partheo** [comment]
Hey, I made the sprites. Allow me to pitch in.

### [the vents] are too eyecandy meaning it's really easy to notice them
Agreed completely. I'm currently working on making them more subtle. These were ported while WIP. Here's a preview(slightly outdated): 


### they are too big
Here's a side-by-side comparison. I guess?
ventsize

### the siphon animation is hard to notice
I was actually trying to improve the fear factor of siphoning scrubbers with this update(using a flashing red), so hearing that it isn't noticeable enough is a big deal. I'll make them more noticeable in my next update. Thanks.

### Only difference is that they're off-center
This is on purpose, so changing this would not be a "fix." The reason I placed them like they are is not so that they line up with the center of the *tile* they're on, but with the lines present on the *floor tiles* they're on. The green lines represent the sprite's center, and the red lines represent the placement of the floor tile lines. So, they are off-center, yes; but floor tiles are also off-center and these were made to be placed on floor tiles.
ventalign

Expect a PR with the updated versions in the future when I finish them.
Also, this PR won't work, please see the changes that I made in the original PR to revert this properly: https://github.com/tgstation/tgstation/pull/32742/files

**Comment by Bawhoppen** [comment]
Oh no, change! Better revert it!

**Comment by ike709** [comment]
While I agree I should have waited until partheo finished the sprites (I didn't even know they were WIP, I stole them from FTL13), I think the WIP sprites are still better than the current sprites.

**Comment by PaznerIV** [comment]
The new vents to me, look ok it just the new scrubbers don't seem to sit right to me I don't know if it the way they look, but it just feels off compare to the vents.

**Comment by Dorsisdwarf** [comment]
I'm fine with newvents, but the new scrubbers just look *wrong*

**Comment by praisenarsie** [comment]
Alright, I'll provide a bit more constructed criticism now.

### Vent
The sprite looks great but it doesn't look like its fits onto the floor right. It looks as if it's not fitted/embedded into the ground and moreso plopped down. Compared those to the old sprites the old vent sprite looks like it is fitted into the ground (possibly due to it's borders/shading or whatever). Also, the fan is too fast. To be honest, the fan's speed should increase/decrease depending on its target pressure. Also, I think the fan can be a bit too noticeable at sometimes and I think it should be more dim.

### Scrubber
Again, the sprite looks great but I'm saying again it doesn't look like it fits on the ground. Also, the siphon animation is too dark/dim as it is kindof hard to notice if it is siphoning. Yes I know you tried to make siphoning look more dangerous but making it a /dark/ red isn't going to make it look more dangerous. It's gonna make it less noticable. Plus the fact that this color is hiding behind that low-grade sun metal symbol in the new scrubber.

### To recap
The sprites don't look like it's fitted/embedded into the ground. The vibrant colors in the sprites look too dark. The fan's speed on the vent should depend on the target pressure. ALSO, as a final point I would like to say that you are changing a STAPLE texture to /tg/ or even ss13 as a whole. Yes I'm aware there were recent changes to computers and also machinery in the past but those were dying for a transition to 3/4. These new vent/scrubber sprites do look good, they look great but I don't think they fit into the old aesthetic in a way.

Apologies if you struggled to understand some of this as I am not the best at explaining things that good. I'm going to tag @WJohn as I believe I saw him talk about these sprites before and he's the 'art director' anyway. I bet he can provide better criticism than me.

🚶

**Comment by Partheo** [comment]
Thanks for the feedback!

As for the vents:
-Fitting it into the floor was definitely something I wanted to try achieving, and I agree it could be done better.
-Fan is too fast/noticeable, yes, that will be fixed. Agreed completely.
-I won't do variable fan speed, unfortunately. Animating multiple fan cycles based on output pressure is a pain.

Scrubber:
-Will make the siphon animated more noticeable. Noted.

I will kindly choose to ignore the "staple" comment. Sorry, but as a spriter this is just impossible to deal with. See: https://github.com/FTL13/FTL13/pull/3026

Has the art director here posted anything I can look at for reference, or is it more of a "yes/no" from him kind of thing?

**Comment by ike709** [comment]
>you are changing a STAPLE texture to /tg/ or even ss13 as a whole

This is literally the worst possible argument for keeping bad sprites. Remember that "staple" sprites used to look like this:

image

Improving the artwork is always a positive change. While arguing about the actual quality of the sprite is okay, don't hate it just because it's different.

**Comment by praisenarsie** [comment]
@ike709 

You misunderstood. Has it occurred to you that I might have a different definition of staple texture or that I couldn't find better wording? Because that's what occurred to me.

First off, that's not /tg/. That's like OS13 or whatever. For all I care our vent sprites probably come from goon but anyway that's beside the point. I mentioned that they are staple textures to /tg/, meaning the players associate it as a well.. familiar. You can see that the old vent sprites (the ones i'm bringing back) are familiar as they are commonly seen in other servers and people are used to that. Plus people didn't even care about the old sprites in a sense that they couldn't really notice them that much when they play. They were unique enough to work and we eventually got used to them. It may seem to ME that I don't like the new sprites we're getting but as a matter of fact I love them but the vent/scrubber sprites Par made feel off and just feels unnatural.

Basically, they're staple textures because they are seen across different servers and we've gotten used to them plus they are SUBTLE. They feel just right probably because we have gotten used to those textures.

**Comment by duncathan** [comment]
The new sprites are better imo

**Comment by WJohn** [comment]
The main problem I have with the new vent sprites is the fan behind the grille is brighter than the grilles themselves. You could probably fix it almost entirely just by having the fan be made darker, and it would stick out less. Having the spinning be proportional to how fast it's pumping out air would also help a lot.

The scrubber LOOKS like a fan, but isn't one, and looks like it'd be easy to stick your foot into. See if you can't rework it to look like something else.

Lastly, I think you could probably rework the lighting on the borders of the scrubbers and vents. Something about them is throwing me off. It's probably that the lighting is towards the bottom, which gives the impression of elevation or a drop after that. See if you can't rework it to look a bit less like they'd stick out of the ground (maybe redo the lighting so it looks like they're set INTO a depression instead of coming out?)

**Comment by WJohn** [comment]
I think it'd be best if we give partheo here a bit of time to try and hammer out some new sprites.

**Comment by Partheo** [comment]
@wjohn this is going to take minimum a week, I'm busy with other projects at the moment. I'm fine if this is merged, I plan to make a PR later anyway.

**Comment by Cyberboss** [comment]
I'll hold you to that
